packages/react-components/src/theme-provider/src/ThemeContext.ts

```diff
@@ -1,8 +1,8 @@
 import { createContext, useContext } from "react";
-import type { ColorScheme } from "./ThemeProvider";
+import type { ColorScheme, Theme } from "./ThemeProvider";
```

Comment:
Potential problem here...

ThemeProvider import ThemeContext and ThemeContext import ThemeProvider

Reply:
Same for useColorScheme